### PR TITLE
docs: Building Simulation Methodology

### PR DESCRIPTION
**Pull request by clackary**
Updates `Benchmarks/BuildingSimulation` README.md with results and methodology for gradient speed, memory utilizations, and power consumption.

### CODE REVIEW

**Comment by clackary** [comment]
Here's a link to aid in review ~since markdown changes aren't rendered here in the PR~: https://github.com/clackary/differentiable-swift-examples/tree/docs/building-simulation-methodology/Benchmarks/BuildingSimulation

(The 'Display in rich diff' button does exactly this)

**Comment by clackary** [comment]
cc @jfilling

**Comment by clackary** [comment]
cc @bjschoenfeld

File: Benchmarks/BuildingSimulation/README.md [hunk 1]
```diff
@@ -90,3 +80,572 @@ and then run the benchmark by going to the `TensorFlow` subdirectory here and us
...
+    <td>17550355200</td>
+    <td>361</td>
+    <td>806886848</td>
+    <td>16</td>
+  </tr>
+</tbody></table>
+
+---
+#### Power Consumption
+In order to measure the energy consumed during program execution we employed the use of a current shunt, a differential probe, and an oscilloscope.
```
Comment (bjschoenfeld) [inline Benchmarks/BuildingSimulation/README.md:578]:
```suggestion
We measured energy consumption during program execution using a current shunt, a differential probe, and an oscilloscope.
```

**Review by bjschoenfeld — APPROVED** [review]

**Review by jfilling — COMMENTED** [review]
We no longer bother to compensate for idle power levels. We have also adjusted the system configuration to guarantee more consistent results (by switching to the performance governor, which removes some of the dynamic frequency variability).

**Review by jfilling — APPROVED** [review]

**Pull request merged**